figgy.go

```diff
@@ -276,3 +287,21 @@ func set(v reflect.Value, s string) error {
 	}
 	return nil
 }
+
+func setJSON(f *field, s string) error {
+	v := f.value
+	if v.Kind() != reflect.Ptr {
+		if !v.CanAddr() {
+			return fmt.Errorf("%s is not addressable", v.Type().String())
+		}
+		v = v.Addr()
+	}
+	if !v.CanInterface() {
+		return fmt.Errorf("%s is not interfaceable", v.Type().String())
+	}
+	if err := json.Unmarshal([]byte(s), v.Interface()); err != nil {
+		fmt.Println(err.Error())
```

Comment:
```suggestion
```

Reply:
Thanks, I accidentally left that in after debugging.